pkg/bitwarden/cli.go

```diff
@@ -28,9 +28,9 @@ func newCliClient(username, password string, addSecret func(s string)) (Client,
 	return newCliClientWithRun(username, password, addSecret, func(args ...string) ([]byte, error) {
 		// bw-password is protected, session in args is not
 		logrus.WithField("args", args).Debug("running bw command ...")
-		out, err := exec.Command("bw", args...).CombinedOutput()
+		out, err := exec.Command("bw", args...).Output()
```

Comment:
Did it fail to stderr on mac because of this?

Reply:
No, this is to stop it from reading stderr